examples/tut-6-rpc/client-go/index.go

```diff
@@ -0,0 +1,214 @@
+package main
+
+import (
+  "fmt"
+  "log"
+  "os"
+  "reflect"
+  "strings"
+  "sync"
+  "time"
+  "encoding/json"
+  "io/ioutil"
+  "math/rand"
+  "net/http"
+)
+
+
+type account   = float64
+type contract  = float64
+type bigNumber = map[string]interface {}
+
+
+func dieIf(err error) {
+  if err != nil {
+    log.Fatal(err)
+    os.Exit(1)
+  }
+}
+
+
+func please(a interface{}, err error) interface{} {
+  dieIf(err)
+  return a
+}
+
+
+func mkRpc() (func(string, ...interface{}) interface{},
+              func(string, contract, map[string]interface{})) {
+
+  proto := "http"
+  host  := os.Getenv("REACH_RPC_SERVER")
+  port  := os.Getenv("REACH_RPC_PORT")
+
+  // TODO wait for server to become available
```

Comment:
It is very annoying that we always need this. I wish there were a good way to build it into Docker

Reply:
Would you rather me investigate a generic Docker solution than redo it in each client language?

Reply:
Naw, it's fine to do it in each one, because when deployed, they won't necessarily be in Docker